network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/wide_table_block.html

```diff
@@ -2,41 +2,31 @@
 {% load wagtailcore_tags table_block_tags %}
 
 {% block block_content %}
-    <div class="article-table-block table-responsive">
+    <div class="article-table-block table-responsive{% if not self.wide %} tw-mx-auto tw-max-w-[80%]{% endif %}">
         {% comment %}
         Modified table template comes from "table_block/blocks/table.html"
         {% endcomment %}
         <table class="table">
-            {% if table_caption %}
-                <caption>{{ table_caption }}</caption>
+            {% if self.table.table_caption %}
```

Comment:
**Minor Suggestion**

I’ve noticed that with the new model definition, we’re referencing `self.table` multiple times throughout the template. While this works perfectly fine, I wanted to mention that we could simplify it by wrapping the template in a `with` statement, like so:

```jinja

{% block block_content %}
       {% with self.table as table %}
...
       {% endwith %}
{% endblock block_content %}

```

That said, I’m totally fine with either approach, so feel free to go with whichever option you think works best!

Reply:
TYVM, great idea 🔥